config/samples/istio-sample-openshift.yaml

```diff
@@ -1,9 +1,10 @@
 apiVersion: operator.istio.io/v1alpha1
 kind: Istio
 metadata:
-  name: istio-sample
+  name: default
 spec:
   version: v1.20.1
   namespace: istio-system
   updateStrategy:
     type: InPlace
+    inactiveRevisionDeletionGracePeriodSeconds: 30
```

Comment:
should we have this field in the samples?

related question: Maybe this field too much implementation detail? What are the use cases users would need to tweak this value?

Reply:
Most of the fields in this sample are there because the OpenShift web console uses these values to pre-populate the Create form.

The `inactiveRevisionDeletionGracePeriodSeconds` is something users will probably want to set themselves, unless we find a good default. We should discuss this in the demo tomorrow.